eo-runtime/pom.xml

```diff
@@ -96,6 +96,7 @@ SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <argLine>@{argLine} -Duser.language=ru -Duser.region=RU</argLine>
+          <argLine>-Xss${stack-size}</argLine>
```

Comment:
@ivan-egorov42 see https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven This `@{argline}` here is for a reason.